tests/unittests/test_error_handling.py

```diff
@@ -72,7 +72,9 @@ def test_request_with_handling_for_400_exceptin_handling(self,mock_send):
         try:
             tap_stream_id = "tap_jira"
             mock_config = {"username":"mock_username","password":"mock_password","base_url": "mock_base_url"}
-            mock_client = http.Client(mock_config)
+            mock_config_path = "mock_config.json"
+            mock_dev_mode = False
+            mock_client = http.Client(mock_config_path, mock_config, mock_dev_mode)
```

Comment:
Can these be pulled out into a separate function/fixture? Looks like we repeat the same code a bunch of times in these tests.

Reply:
Yes. Will do.